main_test.go

```diff
@@ -31,6 +31,13 @@ const TESTDATA = "testdata"
 
 var testTarget = flag.String("target", "", "override test target")
 
+var supportedArches = map[string]string{
```

Comment:
We support other OS/arch combinations too, such as windows/amd64. Maybe we can call this `supportedLinuxArches` or something like that?

Reply:
Yes, good point. I suppose once #2586 is in, this could maybe be expanded, but I'll change this now.

Reply:
> I suppose once https://github.com/tinygo-org/tinygo/pull/2586 is in, this could maybe be expanded, but I'll change this now.

I don't think so: #2586 is only for cross compiling. You can't actually run those binaries on the local host.